src/Servers/Kestrel/Core/src/Middleware/HttpsConnectionMiddleware.cs

```diff
@@ -396,7 +453,7 @@ private static bool IsWindowsVersionIncompatibleWithHttp2()
             {
                 var enableHttp2OnWindows81 = AppContext.TryGetSwitch(EnableWindows81Http2, out var enabled) && enabled;
                 if (Environment.OSVersion.Version < new Version(6, 3) // Missing ALPN support
-                    // Win8.1 and 2012 R2 don't support the right cipher configuration by default.
+                                                                      // Win8.1 and 2012 R2 don't support the right cipher configuration by default.
```

Comment:
The indentation change here seems odd

Reply:
Yea VS wanted this. I don't want to fight it 😆

Reply:
We could just put comments on their own line. We generally do that by convention.